x-pack/plugins/lens/public/xy_visualization/annotations/config_panel/index.tsx

```diff
@@ -0,0 +1,192 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import './index.scss';
+import { i18n } from '@kbn/i18n';
+import { EuiDatePicker, EuiFormRow, EuiSwitch, EuiSwitchEvent, EuiTitle } from '@elastic/eui';
+import type { PaletteRegistry } from 'src/plugins/charts/public';
+import moment from 'moment';
+import { EventAnnotationConfig } from 'src/plugins/event_annotation/common/types';
+import type { VisualizationDimensionEditorProps } from '../../../types';
+import { State, XYState } from '../../types';
+import { FormatFactory } from '../../../../common';
+import { XYAnnotationLayerConfig } from '../../../../common/expressions';
+import { ColorPicker } from '../../xy_config_panel/color_picker';
+import { DimensionEditorSection, NameInput, useDebouncedValue } from '../../../shared_components';
+import { isHorizontalChart } from '../../state_helpers';
+import { MarkerDecorationSettings } from '../../xy_config_panel/shared/marker_decoration_settings';
+import { LineStyleSettings } from '../../xy_config_panel/shared/line_style_settings';
+import { updateLayer } from '../../xy_config_panel';
+import { annotationsIconSet } from './icon_set';
+
+export const defaultAnnotationLabel = i18n.translate('xpack.lens.xyChart.defaultAnnotationLabel', {
+  defaultMessage: 'Event',
+});
+
+export const AnnotationsPanel = (
+  props: VisualizationDimensionEditorProps<State> & {
+    formatFactory: FormatFactory;
+    paletteService: PaletteRegistry;
+  }
+) => {
+  const { state, setState, layerId, accessor } = props;
+  const isHorizontal = isHorizontalChart(state.layers);
+
+  const { inputValue: localState, handleInputChange: setLocalState } = useDebouncedValue<XYState>({
+    value: state,
+    onChange: setState,
+  });
+
+  const index = localState.layers.findIndex((l) => l.layerId === layerId);
+  const localLayer = localState.layers.find(
+    (l) => l.layerId === layerId
+  ) as XYAnnotationLayerConfig;
+
+  const currentAnnotations = localLayer.annotations?.find((c) => c.id === accessor);
+
+  const setAnnotations = useCallback(
+    (annotations: Partial<EventAnnotationConfig> | undefined) => {
+      if (annotations == null) {
+        return;
+      }
+      const newConfigs = [...(localLayer.annotations || [])];
+      const existingIndex = newConfigs.findIndex((c) => c.id === accessor);
+      if (existingIndex !== -1) {
+        newConfigs[existingIndex] = { ...newConfigs[existingIndex], ...annotations };
+      } else {
+        return; // that should never happen because annotations are created before annotations panel is opened
+      }
+      setLocalState(updateLayer(localState, { ...localLayer, annotations: newConfigs }, index));
+    },
+    [accessor, index, localState, localLayer, setLocalState]
+  );
+
+  return (
+    <>
+      <DimensionEditorSection>
+        <EuiTitle size="xs" className="lnsXyConfigHeading">
```

Comment:
To avoid using the same font size as the flyout header and improve the visual hierarchy, can we take the font size down one step for this title?

```suggestion
        <EuiTitle size="xxs" className="lnsXyConfigHeading">
```